hardware/NetworkSwitcher/src/com/yoshino/networkswitcher/NetworkSwitcher.kt

```diff
@@ -57,6 +57,25 @@ class NetworkSwitcher : Service() {
                 "amss_fsg_maple_tar.mbn", "amss_fsg_maple_dsds_tar.mbn")
 
         private const val MODEM_SWITCHER_STATUS = "/cache/modem/modem_switcher_status"
+
+        // Network bitmasks
+        // 2G
+        private const val GSM = TelephonyManager.NETWORK_TYPE_BITMASK_GSM or TelephonyManager.NETWORK_TYPE_BITMASK_GPRS or
+                TelephonyManager.NETWORK_TYPE_BITMASK_EDGE
+
+        private const val CDMA = TelephonyManager.NETWORK_TYPE_BITMASK_CDMA or TelephonyManager.NETWORK_TYPE_BITMASK_CDMA or
```

Comment:
These two constants look identical :p

Reply:
https://github.com/LineageOS/android_frameworks_base/blob/lineage-18.0/telephony/java/android/telephony/RadioAccessFamily.java#L70

Well they are bitmasks and have an OR operation for it. Must be for a reason